examples/media/schema.original.json

```diff
@@ -0,0 +1,295 @@
+            "parameters": {
+              "resource": "media.collections",
+              "method": "post",
+              "required": [
+              "items"
+              ],
+              "properties": {
+                "items": {
+                  "description": "media identifier",
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "format": "uri"
+                  }
+                }
+              }
+            }
+          },
+          "get": {
+            "parameters": {}
+          }
+        },
+        "element": {
+          "post": {
+            "description": "Media can be any media object. All 'encapsulating' media objects like album, playlist, source etc. will be flattend to root level of mediacollection. E.g. if a playlist contains track A and B, the tracks A and B will be added to the mediacollection.",
+            "parameters": {
+              "resource": "media.collections",
+              "method": "post",
+              "required": [
+              "items"
+              ],
+              "properties": {
+                "items": {
+                  "description": "media identifier",
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "format": "uri"
+                  }
+                }
+              }
+            }
+          },
+          "get": {
+            "parameters": {}
+          },
+          "delete": {
+            "parameters": {}
+          }
+        }
+      }
+    },
+    "renderers": {
+      "description": "The media player renderer can be understood as the actual media player. It accepts media objects and applies actions like play, pause etc. to them.",
+      "model": {
+        "name": "rendererObject",
+        "resource": "media.renderers",
+        "properties": {
+          "id": {
+            "description": "renderer id",
+            "type": "string",
+            "format": "uuid"
+          },
+          "name": {
+            "description": "renderer name",
+            "type": "string"
+          },
+          "uri": {
+            "description": "object uri",
+            "type": "string",
+            "format": "uri"
+          },
+          "media": {
+            "description": "current media collection",
+            "type": "object",
+            "oneOf": [
+            {
+              "#ref": "media.collections.mediaCollectionObject"
+            }
+            ]
+          },
+          "mediaindex": {
+            "description": "DEPRECATED: index of actual media object if media is a collection (starting index is 0)",
+            "type": "integer",
+            "minimum": 0
+          },
```

Comment:
As this is a complete "rewrite" you should remove this property and not mark it as deprecated.